<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>IP address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>66</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>135</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="portEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>139</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ConnectBtn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>LED on/off</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QCheckBox" name="led21">
      <property name="text">
       <string>LED 21</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="led22">
      <property name="text">
       <string>LED 22</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="led23">
      <property name="text">
       <string>LED 23</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>servo</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="degree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="move">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>jo</string>
    </property>
    <widget class="QLineEdit" name="sensor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
